cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectBuild.java

```diff
@@ -322,7 +320,7 @@ public Pod generateBuilderPod(boolean isOpenShift, ImagePullPolicy imagePullPoli
 
         List<String> args = additionalKanikoOptions != null ? new ArrayList<>(4 + additionalKanikoOptions.size()) : new ArrayList<>(4);
         args.add("--dockerfile=/dockerfile/Dockerfile");
-        args.add("--context=dir://workspace");
+        //args.add("--context=dir://workspace");
```

Comment:
Do you plan to remove it or leaving it as a comment does have any intention?

Reply:
Sorry, my mistake - I deleted the line. Thanks for pointing it out.